src/components/tasks/card/index.tsx

```diff
@@ -214,17 +215,14 @@ const Card: FC<Props> = ({
         );
     }
 
-    const onEditEnabled = () => {
-        actions.onEditRoute();
-    };
     const EditButton = () => (
         <div className={classNames.editButton} data-testid="edit-button">
             <Image
                 src="/pencil.webp"
                 alt="edit"
```

Comment:
please fix this text

Reply:
please suggest what should be alt instead

Reply:
pencil icon to represent edit button  
link if you want to read about it : [Link](https://accessibility.huit.harvard.edu/describe-content-images)

Reply:
done